build-logic/publication/src/main/kotlin/com/avito/android/publish/AndroidLibraryPublishExtension.kt

```diff
@@ -12,7 +12,6 @@ abstract class AndroidLibraryPublishExtension @Inject constructor(objects: Objec
      *
      * default: release
      */
-    @Suppress("UnstableApiUsage")
```

Comment:
I'd love to see such massive changes in a separate PR.
Separate commits don't solve all inconveniences.

Reply:
Ok, i'll keep that in mind;
i thought it was just a boyscout refactoring, like remove some meta info from files

Reply:
done